tutorial/TensorFlow_Keras_EfficientNet.ipynb

```diff
@@ -70,36 +70,21 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    },
     "scrolled": true
    },
-   "outputs": [
-    {
-     "name": "stderr",
-     "output_type": "stream",
-     "text": [
-      "Using TensorFlow backend.\n"
-     ]
-    },
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "WARNING:tensorflow:Large dropout rate: 0.5125 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.\n",
-      "WARNING:tensorflow:Large dropout rate: 0.525 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.\n",
-      "WARNING:tensorflow:Large dropout rate: 0.5375 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.\n",
-      "WARNING:tensorflow:Large dropout rate: 0.55 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.\n",
-      "WARNING:tensorflow:Large dropout rate: 0.5625 (>0.5). In TensorFlow 2.x, dropout() uses dropout rate instead of keep_prob. Please ensure that this is intended.\n"
-     ]
-    }
-   ],
+   "outputs": [],
    "source": [
+    "import numpy as np\n",
     "import efficientnet.tfkeras as efn\n",
-    "from keras.applications.imagenet_utils import decode_predictions\n",
-    "from efficientnet.keras import center_crop_and_resize, preprocess_input\n",
+    "from tensorflow.keras.applications.imagenet_utils import decode_predictions, preprocess_input\n",
+    "from efficientnet.preprocessing import center_crop_and_resize\n",
     "from skimage.io import imread\n",
-    "model = efn.EfficientNetB7(weights='imagenet')"
+    "model = efn.EfficientNetB0(weights='imagenet')"
```

Comment:
Not using EfficientNetB7?

Reply:
too slow for user trying B7 end to end. Given it's a tutorial, it should be simple and fast.